ElliotForWater/efw-webapp
# feature: server-side rendering with user state from cookies

## Description
"User state" is persisted in cookies client-side and read before rendering client-side (but not server-side). This leads to the initial HTML being slightly different than what the client renders and leads to visiable delayed updates on initial app load (or browser refreshes).

Naturally, cookies are also sent to the server with every request so this PR enables using the cookie values on the server as well via the request data.

#### Impacted Areas in Application
Server-side rendering and initial display of number of searches.


#### Steps to Test or Reproduce
1. Search (see number of searches go to > 0)
2. Refresh page

Old version you see first "0", then the number a bit delayed.

------------------------------------------------
## PR Checklist:
